ansible/roles/cli/tasks/download_cli.yml

```diff
@@ -10,7 +10,7 @@
     mode=0755
     validate_certs=False
     force=True
-  when: "'environments/docker-machine' not in inventory_dir"
+  when: "'environments/docker-machine' and 'environments/mac' not in inventory_dir"
```

Comment:
I think this line should be
  when: "'environments/docker-machine' not in inventory_dir and 'environments/mac' not in inventory_dir".